backend/api/main.py

```diff
@@ -167,6 +170,69 @@ def update_moodle_settings(
     return updated_moodle_settings
 
 
+@app.post("/moodle/course/build", dependencies=[Depends(verify_manager)])
```

Comment:
Since we're using the dependency as an input, we can actually remove it here and just apply the right role below:

```python
@app.post("/moodle/course/build")
def create_course_build(
    current_user: Annotated[dict, Depends(verify_manager)],
    course_build_settings: BaseCourseBuildSettings,
    db: Session = Depends(get_db),
) -> FullCourseBuildSettings:
```